### ISSUE TITLE
python error from pip version

### ISSUE THREAD

**Issue by dmaglio** [andialbrecht/sqlparse#313]
```
Traceback (most recent call last):
  File "/usr/local/bin/sqlformat", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/cli.py", line 165, in main
    s = sqlparse.format(data, **formatter_opts)
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/__init__.py", line 60, in format
    return u''.join(stack.run(sql, encoding))
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/engine/filter_stack.py", line 39, in run
    filter_.process(stmt)
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/filters/aligned_indent.py", line 128, in process
    self._process(stmt)
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/filters/aligned_indent.py", line 125, in _process
    func(tlist)
  File "/usr/local/lib/python3.5/dist-packages/sqlparse/filters/aligned_indent.py", line 41, in _process_statement
    if tlist.tokens[0].is_whitespace and self.indent == 0:
IndexError: list index out of range
```
v0.2.2 from pip in ubuntu 16.04 and try to format a sql file

**Comment by datarookery** [comment]
Reproduced using align (-a) option on file with statement ended in terminator `;` followed by newline (lf || cr lf). No error if: `\n` is removed or any character including `\n` exists on next line or `;` not followed by `\n`.

**Comment by lassik** [comment]
Probably fixed by PR #390 but that version is not yet available from `pip`?

Here's a very short way to reproduce (what I think is) the same bug:

    echo 'select from foo;' | sqlformat -a -

**Comment by andialbrecht** [comment]
You're right. This was fixed in #390 and will be in 0.3.0. I'm about to release this version pretty soon.
